packages/highlexer/highlexer.0.1/opam

```diff
@@ -0,0 +1,33 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "An OCaml syntax highlighter based on the OCaml lexer"
+description: "An OCaml syntax highlighter based on the OCaml lexer"
+maintainer: ["Emile Trotignon"]
+authors: ["Emile Trotignon"]
+license: "MIT"
+tags: ["js" "syntax" "highlighting"]
+homepage: "https://github.com/EmileTrotignon/highlexer"
+doc: "https://ocaml.org/p/highlexer/latest/doc/index.html"
+bug-reports: "https://github.com/EmileTrotignon/highlexer/issues"
+depends: [
+  "ocaml" {>= "4.10"}
+  "dune" {>= "3.11"}
+  "menhir" {>= "20230608"}
+  "alcotest" {with-test}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/EmileTrotignon/highlexer.git"
```

Comment:
```suggestion
dev-repo: "git+https://github.com/EmileTrotignon/highlexer.git"
url {
  src: "https://github.com/EmileTrotignon/highlexer/archive/refs/tags/0.1.tar.gz"
  checksum: [
    "md5=5e6b2b4fa38383e0b3bc3a96be929a66"
    "sha256=9146a570641406dd8325682f57ce6062ad770f2dbe1fe5180ff607171e69bee2"
  ]
}
```
Thanks to @kit-ty-kate we found out the mystery! There is no archive here!

Reply:
Thanks a lot ! That's indeed an issue.